docs/frameworks/expressjs.md

```diff
@@ -343,7 +234,7 @@ You can find this URL in the deployment output under the `App Dashboard URL` sec
 
 ### Do I Need to Modify My Express App Code?
 
-Your Express project will only require the above [adjustments](#deployment-guide), as long as it is written in a **stateless** manner. To find out more about [stateless vs. stateful follow this link](https://stackoverflow.com/questions/5329618/stateless-vs-stateful).
+Your Express.js project will only require the above [adjustments](#deployment-guide), as long as it is written in a **stateless** manner. To find out more about [stateless vs. stateful follow this link](https://stackoverflow.com/questions/5329618/stateless-vs-stateful).
```

Comment:
This was related to `serverless-http` - we now support deploying Express apps without any code changes.

You can either remove this `### Do I Need to Modify My Express App Code?` section or the answer is "nope, you don't need code changes`.